code/src/index.css

```diff
@@ -1,13 +1,189 @@
 body {
   margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
-    "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
+  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen',
+    'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue',
     sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
 }
 
+html,
+body {
+  height: 100%;
+  min-height: 100%;
+}
+
 code {
-  font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
+  font-family: source-code-pro, Menlo, Monaco, Consolas, 'Courier New',
     monospace;
 }
+
+.outer-wrapper {
+  padding: 120px 10px;
+  background-color: #0e0e0e;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.inner-wrapper {
+  background-color: #181818;
+  padding: 15px;
+  width: 80%;
+  margin: 0 auto;
+  display: flex;
+  flex-wrap: wrap;
+  max-width: 1200px;
+}
+
+.albums-wrapper {
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr 1fr;
+}
+
+.album-container {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start; /* Makes it so the link is not the full width */
+  padding: 20px 10px;
+}
+
+.title {
+  font-size: 18px;
+  color: white;
+  width: 100%;
+  margin: 0;
+  padding-bottom: 10px;
+  border-bottom: 1px solid rgb(44, 44, 44);
+}
+
+.album-name {
+  font-family: Helvetica, sans-serif;
+  color: white;
+  text-decoration: none;
+  font-size: 14px;
+  margin: 2px 0;
+}
+.album-name:hover {
+  text-decoration: underline;
+}
+
+.artist-name {
+  font-size: 14px;
+  font-family: Helvetica, sans-serif;
+
+  margin: 0;
+  color: #a0a0a0;
+  text-decoration: none;
+}
+
+/* Add a comma before every child that is not the first nor last. */
+.artist-name-container > :not(:last-child):not(:first-child):before {
+  content: ', ';
+  white-space: pre;
```

Comment:
Is this making space between arist name and the comma? Or is that done in line 82? Or what is this doing?